api/v1/design/resource.go

```diff
@@ -102,6 +102,7 @@ var _ = Service("resource", func() {
 
 			Response(StatusOK)
 			Response("internal-error", StatusInternalServerError)
+			Response("not-found", StatusNotFound)
```

Comment:
Just a confirmation, since we are adding it in design now, in the service are returning the `not-found` error

Reply:
yes, it returns

Reply:
Could you please point where it is been used ?

Reply:
in case db has no resource , earlier resource api was returning 400 status code , so now it returns 404 status code , these status code  are used for api error